electrum/gui/qt/email_notification_dialogs.py

```diff
@@ -245,42 +308,35 @@ def run_notification(self):
             what_next = self.run_single_view(self._choose_email)
             if what_next == self.State.NEXT:
                 EmailNotificationConfig.save_email_to_config(self.config, self.wallet, self._email)
-                what_next = self.run_single_view(self.confirm_pin, _('Back'))
-            elif what_next == self.State.SHOW_EMAIL_SUBSCRIBED:
-                self.show_message(self._error_message)
-                break
+                what_next = self.run_single_view(self.confirm_pin, back_button_name=_('Back'))
             else:
                 break
 
         if what_next in [self.State.NEXT, self.State.SHOW_EMAIL_SUBSCRIBED]:
+            message = (
+                    self._error_message + '\n' +
+                    _('You have successfully added your email address.')
+            ) if what_next == self.State.SHOW_EMAIL_SUBSCRIBED else _('You have successfully subscribed wallet')
```

Comment:
Is there a way to write this in cleaner way?

Reply:
Done